google_api_lib/tasks.py

```diff
@@ -78,8 +78,7 @@ def transfer_ownership(self, file, template_file_id) -> None:
     self.drive_service().permissions().update(
         fileId=file["id"],
         permissionId=permission["id"],
-        body={"role": "owner"},
-        transferOwnership=True,
+        body={"role": "writer", "pendingOwner": "true"},
```

Comment:
any reason for changing role from "owner" to "writer"?

Reply:
"owner" is no longer allowed, have to do pendingOwner instead, but role is still a required field, so =/